services/src/main/java/org/keycloak/services/resources/account/AccountCredentialResource.java

```diff
@@ -57,6 +57,139 @@ public AccountCredentialResource(KeycloakSession session, EventBuilder event, Us
+        
+        List<CredentialModel> passwordUserCreds = new java.util.ArrayList<>();
+        passwordUserCreds.add(models.get(0));
+        
+        List<CredentialModel> otpUserCreds = new java.util.ArrayList<>();
+        if (models.size() > 1) otpUserCreds.add(models.get(1));
+        if (models.size() > 2) otpUserCreds.add(models.get(2));
+        
+        List<CredentialModel> webauthnUserCreds = new java.util.ArrayList<>();
+        CredentialModel webauthnCred = new CredentialModel();
+        webauthnCred.setId("bogus-id");
+        webauthnCred.setUserLabel("yubikey key");
+        webauthnCred.setCreatedDate(1579122652382L);
+        webauthnUserCreds.add(webauthnCred);
+        
+        List<CredentialModel> webauthnStrongUserCreds = new java.util.ArrayList<>();
+        CredentialModel webauthnStrongCred = new CredentialModel();
+        webauthnStrongCred.setId("bogus-id-for-webauthnStrong");
+        webauthnStrongCred.setUserLabel("My very strong key with required PIN");
+        webauthnStrongCred.setCreatedDate(1579122652382L);
+        webauthnUserCreds.add(webauthnStrongCred);
+        
+        CredentialContainer password = new CredentialContainer(
+                                        "password",
+                                        "password",
+                                        "passwordTooltip",
+                                        true,
+                                        null, // no create action
+                                        "UPDATE_PASSWORD",
+                                        false,
+                                        passwordUserCreds
+                                        );
+        CredentialContainer otp = new CredentialContainer(
+                                        "two-factor",
+                                        "otp",
+                                        "otpTooltip",
+                                        true,
+                                        "CONFIGURE_TOTP", 
+                                        null, // no update action
+                                        true,
+                                        otpUserCreds
+                                        );
+        CredentialContainer webAuthn = new CredentialContainer(
+                                        "two-factor",
+                                        "webauthn",
+                                        "webauthntooltip",
+                                        true,
+                                        "CONFIGURE_WEBAUTHN", 
+                                        null, // no update action
+                                        true,
+                                        webauthnUserCreds
+                                        );
+        CredentialContainer passwordless = new CredentialContainer(
+                                        "passwordless",
+                                        "webauthn-strong",
```

Comment:
The credential will be named "Webauthn Passwordless" instead of "WebAuthn Strong" . For more details, see the keycloak-dev thread "Strong WebAuthn for passwordless authentication" and the PR https://github.com/keycloak/keycloak/pull/6649 .

Reply:
Right now, the UI just uses "webauthn-strong" as a localization key.  So it passes "webauthn-strong" and gets "WebAuthn Strong" as a result in English.

Perhaps you just want to change this so that the REST API returns "webauthn-passwordless" and it shows "Webauthn Passwordless" in the UI?

Reply:
Updated the UI.  See new pic above.

Reply:
Thanks!